buildlib/az-helpers.sh

```diff
@@ -214,15 +214,26 @@ check_release_build() {
     build_reason=$1
     build_sourceversion=$2
     title_mask=$3
+    launch=False
 
+    # DRP release scheduled testing
+    if [ "${build_reason}" == "Schedule" ]; then
+        launch=True
```

Comment:
shouldn't we check *DRP match under this branch?

Reply:
The scheduler is only set for the DRP pipeline, but it's better to be on the safe side.